wrappers/services.go

```diff
@@ -608,6 +646,38 @@ func EnsureSnapServices(snaps map[*snap.Info]*SnapServiceOptions, opts *EnsureSn
 		}
 	}
 
+	handleSliceModification := func(path string, content []byte) error {
```

Comment:
we might want to call the obvserve callback with appropriate params, or at least leave a TODO about that

Reply:
done